i18n/fr-FR.yml

```diff
@@ -348,6 +373,11 @@ common:
     noItineraryToDisplay: Aucun trajet à afficher.
     transfers: "{transfers, plural, =0 {} one {{transfers} correspondance} other {{transfers} correspondances}}"
 
+  # Standard navigation
+  navigation:
+    back: Retour
+    startOver: Recommencer
```

Comment:
See comments for English file.

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/454/commits/983e90747ffa698aa8b1547fc31e776fe040101e